test/integration/test_classification_accuracy_semantic_robustness.py

```diff
@@ -236,151 +164,10 @@ def test_evaluate_sample(self, casr_test_case):
                     },
                 },
             ),
-            CASRTestCase(
```

Comment:
I deleted two of the test cases, as we are already unit-testing all of the different perturbation variants. We've already done this for general semantic robustness.